src/components/Logo/Logo.stories.tsx

```diff
@@ -1,9 +1,17 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import Logo from './Logo';
+import { BrowserRouter } from 'react-router-dom';
 
 export default {
   component: Logo,
   title: 'Components/Logo',
+  decorators: [
+    Story => (
+      <BrowserRouter>
```

Comment:
Please add decorator globally instead of just for this component

Reply:
Fixed